lib/classes/TradeOffer.js

```diff
@@ -399,60 +410,65 @@ TradeOffer.prototype.send = function(callback) {
 			return;
 		}
 
-		if (body && this.state == ETradeOfferState.CreatedNeedsConfirmation) {
+		if (body && this.state === ETradeOfferState.CreatedNeedsConfirmation) {
 			callback(null, 'pending');
 		} else if (body && body.tradeofferid) {
 			callback(null, 'sent');
 		} else {
-			callback(new Error("Unknown response"));
+			callback(new Error('Unknown response'));
 		}
-	}, "tradeoffermanager");
+	}, 'tradeoffermanager');
 };
 
-TradeOffer.prototype.cancel = TradeOffer.prototype.decline = function(callback) {
+// eslint-disable-next-line no-multi-assign
+TradeOffer.prototype.cancel = TradeOffer.prototype.decline = function (callback) {
 	if (!this.id) {
-		Helpers.makeAnError(new Error("Cannot cancel or decline an unsent offer"), callback);
+		Helpers.makeAnError(new Error('Cannot cancel or decline an unsent offer'), callback);
 		return;
 	}
 
-	if (this.state != ETradeOfferState.Active && this.state != ETradeOfferState.CreatedNeedsConfirmation) {
-		Helpers.makeAnError(new Error(`Offer #${this.id} is not active, so it may not be cancelled or declined`), callback);
+	if (this.state !== ETradeOfferState.Active && this.state !== ETradeOfferState.CreatedNeedsConfirmation) {
+		Helpers.makeAnError(new Error(`Offer #${this.id} is not active, \
+so it may not be cancelled or declined`), callback);
```

Comment:
Not a fan of any situation which ends up with code starting at the very beginning of the line, despite indentation.